web/test-data/test_gtfs_files.txt

```diff
@@ -0,0 +1,4 @@
+f-9qc-fairfield~ca~us.zip
```

Comment:
There's no rhyme or reason to the feeds I chose, I just picked ones that were relatively close to each other + were for areas firmly included in the bbox (I didn't formally check how the routes from these feeds overlap the bboxes, but I'm not sure it matters. We're currently only requesting routes far inside the bbox itself)

Reply:
Cool. That sounds good for now. It seems easy enough to update as we go forward so i think this is fine for now